### ISSUE TITLE
Filter panel visual improvements

### ISSUE THREAD

**Issue by nickvasilyev** [lucidworks/banana#54]
Most of the business users that see the dashboard report that the number one eye sore is the Filters panel. Is there a way you guys can make it a bit more visually appealing? 

Some of the things I noticed:
- Date filtering puts in a UTC date, which is not something that most users understand. I have to have two date fields, one for human friendly dates for display in panels like table view and a second field for UTC date for banana. If you guys can massage the UTC date into localtime across the dashboard it would be great. 
- I see cases where text is URL encoded if I am filtering on certain fields, for example spaces show up as %20.

**Comment by draytoncharles** [comment]
Agreed except for UTC. It is pretty valuable for distributed teams,
particularly investigators, because it allows them to more easily
coordinate timestamps.

I'm on board with two times.

**Comment by nickvasilyev** [comment]
There are definitely merits in having a UTC timestamps, however most users get totally lost with it. I think it should follow the same conversion that is in the histogram panel, so the values are on the same scale and displayed in the same format.
